google-apis-core/lib/google/apis/core/api_command.rb

```diff
@@ -18,6 +18,7 @@
 require 'google/apis/errors'
 require 'json'
 require 'retriable'
+require "securerandom"
```

Comment:
Do we actually need the stronger duplicate avoidance characteristics of SecureRandom? It does make a performance difference.

Reply:
Well, grumble, it looks like Ruby 2.5 has `SecureRandom.uuid` but not `Random.uuid`. I guess that's an argument for keeping SecureRandom.

Reply:
Yeah 😅  , I'll move back to `SecureRandom`